test/JDBC/expected/BABEL-2349.out

```diff
@@ -39,6 +39,45 @@ int
 use master;
 go
 
+-- sysindexes should also exist in dbo schema
+-- If there are white spaces between schema name and catalog name then those need to be ignored
+-- case insensitive check
+SELECT COUNT(*) FROM sys.    SySInDeXes where id=OBJECT_ID('t1');
+go
+~~START~~
+int
+2
+~~END~~
+
+
+SELECT COUNT(*) FROM dbo.    SySInDeXes where id=OBJECT_ID('t1');
+go
+~~START~~
+int
+2
+~~END~~
```

Comment:
What if we execute `SELECT COUNT(*), 'dbo.sysindexes' FROM dbo.    SySInDeXes where id=OBJECT_ID('t1');` ? Will only replace the second instance? We should have a testcase for that.